metering/ingest/s3_client.py

```diff
@@ -37,12 +37,28 @@ def send(self, payload):
         Create meter records with `metering.ingest.create_ingest_payload`.
         """
 
-        data = json.dumps(payload)
-
         file_name = "{}-{}.json".format(
             uuid4(), datetime.now().strftime(r"%d-%b-%Y-%H-%M-%S-%f")
         )
 
+        return self._putObject(file_name, payload)
+
+    def sendCustom(self, payload):
+        """
+        Uploads the payload to S3.
+
+        The payload format can be arbitrary. Events will be parsed using custom schemas defined for the account.
+        """
+
+        file_name = "ingest/schema_detection=AUTO/{}-{}.json".format(
+            uuid4(), datetime.now().strftime(r"%d-%b-%Y-%H-%M-%S-%f")
+        )
+
+        return self._putObject(file_name, payload)
+
+    def _putObject(self, file_name, payload):
+        data = json.dumps(payload)
+
         response = self.s3.Object(self.bucket_name, file_name).put(Body=data)
 
         status_code = response["ResponseMetadata"]["HTTPStatusCode"]
```

Comment:
I believe this check is unnecessary... the s3 client would throw instead. Could you write a todo to review this error handling? also remove (replace) the fixme below